app/src/main/java/org/xwiki/android/sync/activities/BaseViewFlipper.java

```diff
@@ -29,22 +30,44 @@
  * BaseViewFlipper
  */
 public abstract class BaseViewFlipper {
+    /**
+     * Activity for operations
+     */
     protected AuthenticatorActivity mActivity;
+    /**
```

Comment:
You should have an empty line between each field.

Reply:
Between this fields was not new lines and for this reason they was not added for javadocs, but I can fix it